operator/manifests/crds/core.observatorium.io_observatoria.yaml

```
@@ -179,6 +179,9 @@ spec:
             compact:
               description: Thanos CompactSpec
               properties:
+                enableDownSampling:
```

nit, if we are treating `downsampling` as one single word, then the casing should be:
```suggestion
                enableDownsampling:
```

---

sorry for missing the previous comment to use `downsampling` as a single word.